docs/release-notes/release-notes-0.18.1.md

```diff
@@ -19,6 +19,9 @@
 
 # Bug Fixes
 
+* [Fixed a bug](https://github.com/lightningnetwork/lnd/pull/7867#issuecomment-2158279005)
+  that caused LND to read the config only partially and continued with the startup
```

Comment:
nit: the release notes needs to contain a link to the PR as well, otherwise the CI script doesn't count that step as resolved. Also, please finish the sentence with a full stop.

Reply:
Ah, I was supposed to link the PR, not the issue. Got it. Fixed :v: